api-spec/extensions/fields/README.md

```diff
@@ -2,29 +2,96 @@
 
 **Extension [Maturity Classification](../../../extensions/README.md#extension-maturity): Pilot**
```

Comment:
Is all the semantics below - whenever possible - also modeled in the OpenAPI? Especially the content types?

Reply:
I think they are, though it's very limited.  This behavior follows the description in one of my other PRs of using simple string parameters for GET or POST form, and additional attributes into the JSON body for POST application/json.